app/src/main/res/values/strings.xml

```diff
@@ -276,71 +277,71 @@
     <string name="settings_general_header">General</string>
     <string name="settings_general_default_base_url_key">DefaultBaseURL</string>
     <string name="settings_general_default_base_url_title">Default server</string>
-    <string name="settings_general_default_base_url_message">To use your own server as a default when subscribing to new topics and/or sharing to topics, enter the server base URL.</string>
+    <string name="settings_general_default_base_url_message">Enter its base URL to use your own server as a default when subscribing to new topics and/or sharing to topics.</string>
     <string name="settings_general_default_base_url_default_summary">%1$s (default)</string>
     <string name="settings_general_users_key">ManageUsers</string>
     <string name="settings_general_users_title">Manage users</string>
     <string name="settings_general_users_summary">Add/remove users for protected topics</string>
     <string name="settings_general_users_prefs_title">Users</string>
     <string name="settings_general_users_prefs_user_not_used">Not used by any topics</string>
-    <string name="settings_general_users_prefs_user_used_by_one">Used by topic %1$s</string>
-    <string name="settings_general_users_prefs_user_used_by_many">Used by topics %1$s</string>
+    <string name="settings_general_users_prefs_user_used_by_one">Used by \"%1$s\" topic</string>
+    <string name="settings_general_users_prefs_user_used_by_many">Used by \"%1$s\" topics</string>
     <string name="settings_general_users_prefs_user_add">Add users</string>
     <string name="settings_general_users_prefs_user_add_title">Add new user</string>
     <string name="settings_general_users_prefs_user_add_summary">Create a new user for a new server</string>
     <string name="settings_general_dark_mode_key">DarkMode</string>
     <string name="settings_general_dark_mode_title">Dark mode</string>
-    <string name="settings_general_dark_mode_summary_system">Use the system default</string>
-    <string name="settings_general_dark_mode_summary_light">Light mode is enabled</string>
-    <string name="settings_general_dark_mode_summary_dark">Dark mode is enabled. Are you a vampire?</string>
+    <string name="settings_general_dark_mode_summary_system">Using the system default</string>
+    <string name="settings_general_dark_mode_summary_light">Light mode on</string>
+    <string name="settings_general_dark_mode_summary_dark">Dark mode on. Are you a vampire?</string>
     <string name="settings_general_dark_mode_entry_system">Use system default</string>
     <string name="settings_general_dark_mode_entry_light">Light mode</string>
     <string name="settings_general_dark_mode_entry_dark">Dark mode</string>
-    <string name="settings_backup_restore_header">Backup &amp; Restore</string>
+    <string name="settings_backup_restore_header">Backups and restoration</string>
```

Comment:
No :-(

Reply:
The ampersand is incorrect is the issue to solve.